bibliopixel/gamma.py

```diff
@@ -17,7 +17,7 @@ def get(self, i):
 # From https://github.com/scottjgibson/PixelPi/blob/master/pixelpi.py
 APA102 = Gamma(gamma=2.5, offset=0.5)
 WS2801 = SM16716 = Gamma(gamma=2.5)
-DEFAULT = Gamma()
+NONE = DEFAULT = Gamma()  # For when you need no correction
```

Comment:
I'm treating it like this one:
```WS2812B = NEOPIXEL = WS2812 = Gamma(gamma=1 / 0.45)```
There are different ways of looking at it and I'd rather have multiple that mean the same thing so that people could use whatever makes logical sense to them.